### ISSUE TITLE
Module not found in 1.0.9

### ISSUE THREAD

**Issue by guillaumebriday** [lukaszflorczak/vue-agile#89]
Hey !

I got errors in v1.0.9 (don't know if it's related) 

```
Module not found: Error: Can't resolve '@/mixins/handlers'
Module not found: Error: Can't resolve '@/mixins/props'
Module not found: Error: Can't resolve '@/mixins/helpers'
...
```

And so on for each mixins

Do you know how to fix it ?

Thank you !

**Comment by guillaumebriday** [comment]
And it works fine in 1.0.8 !

**Comment by lukaszflorczak** [comment]
Hi. I published fix in `v1.0.10`. It should be OK now

**Comment by guillaumebriday** [comment]
It works ! Thank you @lukaszflorczak
